test/build/plugin-vue/plugin-vue.test.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+const path = require('path');
+const {setupBuildTest} = require('../../test-utils');
+
+const cwd = path.join(__dirname, 'build');
+
+describe('@snowpack/plugin-vue', () => {
+  beforeAll(() => {
+    setupBuildTest(__dirname);
+  });
+
+  it('assumes .vue.js', () => {
+    const index = fs.readFileSync(path.join(cwd, '_dist_', 'index.vue.js'), 'utf8');
+    expect(index).toEqual(
+      expect.stringContaining(`import MyComponent from './MyComponent.vue.js'`),
```

Comment:
This test asserts that `MyComponent` is properly suffixed with `.vue` rather than `.js`.